nomad/job_endpoint.go

```diff
@@ -312,10 +312,31 @@ func (j *Job) Register(args *structs.JobRegisterRequest, reply *structs.JobRegis
 		return err
 	}
 
+	// Create a new evaluation
+	now := time.Now().UTC().UnixNano()
```

Comment:
UnixNano is always in nanoseconds since the Unix epoch which is in UTC, so we don't need to explicitly convert to UTC. No big deal, same result either way.

```suggestion
	now := time.Now().UnixNano()
```